CHANGELOG.md

```diff
@@ -18,6 +18,9 @@ and this project adheres to [Semantic Versioning](http://semver.org/).
 - Consolidated configuration management of `UIConfig` from components into `UIManager`
 - Configuration from the player source now takes precedence over the configuration passed into the `UIManager`
 
+### Fixed
+- IE & Firefox could leave active/hovered select boxes floating after the parent container was hidden.
```

Comment:
Please change to "... could leave the dropdown panel of a `SelectBox` floating ..." (mind the component name in code tags) and remove the trailing "."

Reply:
that was actually intended as i operated directly on the native html element, but as i will have to change that too, this request makes sense :)